modules/formulize/include/formdisplay.php

```diff
@@ -49,155 +49,178 @@
 
 // NEED TO USE OUR OWN VERSION OF THE CLASS, TO GET ELEMENT NAMES IN THE TR TAGS FOR EACH ROW
 class formulize_themeForm extends XoopsThemeForm {
-    

```

Comment:
I'm not sure why apparently the same code is showing as a removal and then an addition in this file? Is that a whitespace issue? Check that these methods in side the formulize_themeForm class have not actually changed, or if they have, isolate the changes to just the lines that have changed.

Reply:
There are differences between the current iteration and the UCOSP branch for the formulize_themeForm class. I'm not sure why it only marked the line here though.